src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -0,0 +1,1100 @@
+            equipmentSectionIdDefaultValue:
+                formValues?.phaseTapChanger?.regulatingTerminalId ?? '',
+        });
+
+    const [phaseLowTapPosition, phaseLowTapPositionField] = useDoubleValue({
+        label: 'LowTapPosition',
+        validation: {
+            isFieldRequired: phaseTapChangerEnabled,
+            isFieldNumeric: true,
+        },
+        inputForm: phaseTapInputForm,
+        defaultValue: formValues?.phaseTapChanger?.lowTapPosition,
+        formProps: { disabled: !phaseTapChangerEnabled },
+    });
+
+    const [phaseHighTapPosition, phaseHighTapPositionField] = useDoubleValue({
+        label: 'HighTapPosition',
+        validation: {
+            isFieldRequired: phaseTapChangerEnabled && !editData && !isCopy,
+            isFieldNumeric: true,
+        },
+        inputForm: phaseTapInputForm,
+        formProps: { disabled: !phaseTapChangerEnabled },
+    });
+
+    const [phaseTapPosition, phaseTapPositionField] = useDoubleValue({
+        label: 'TapPosition',
+        validation: {
+            isFieldRequired: phaseTapChangerEnabled,
+            isFieldNumeric: true,
+            isValueGreaterThan: phaseLowTapPosition - 1,
+            isValueLessOrEqualTo: phaseHighTapPosition,
+            errorMsgId: 'TapPositionBetweenLowAndHighTapPositionValue',
+        },
+        inputForm: phaseTapInputForm,
+        defaultValue: formValues?.phaseTapChanger?.tapPosition,
+        formProps: { disabled: !phaseTapChangerEnabled },
+    });
+
+    //Copy data from existing equipment to form fields
+    const toFormValues = (twt) => {
+        if (twt.ratioTapChanger?.ratioTapSteps) {
+            setRatioTapRows(
+                Object.values(twt.ratioTapChanger?.ratioTapSteps).map(
+                    (step) => {
+                        return {
+                            key: step.index,
+                            tap: step.index,
+                            resistance: step.r,
+                            reactance: step.x,
+                            conductance: step.g,
+                            susceptance: step.b,
+                            ratio: step.rho,
+                        };
+                    }
+                )
+            );
+        }
+
+        if (twt.phaseTapChanger?.phaseTapSteps) {
+            setPhaseTapRows(
+                Object.values(twt.phaseTapChanger?.phaseTapSteps).map(
+                    (step) => {
+                        return {
+                            key: step.index,
+                            tap: step.index,
+                            resistance: step.r,
+                            reactance: step.x,
+                            conductance: step.g,
+                            susceptance: step.b,
+                            ratio: step.rho,
+                            alpha: step.alpha,
+                        };
+                    }
+                )
+            );
+        }
+
+        setIsCopy(true);
+        return {
```

Comment:
Ouch

Reply:
There's a whole lot of properties that's for sure